node/src/components/consensus/protocols/highway.rs

```diff
@@ -710,4 +710,29 @@ where
     fn instance_id(&self) -> &C::InstanceId {
         self.highway.instance_id()
     }
+
+    fn recreate_timers(&self) -> Vec<ProtocolOutcome<I, C>> {
+        let mut outcomes = vec![ProtocolOutcome::ScheduleTimer(
+            Timestamp::now(),
+            TIMER_ID_PURGE_VERTICES,
+        )];
```

Comment:
I'm adding another timer in https://github.com/CasperLabs/casper-node/pull/927.
Would be good to deduplicate this code, so that we can't forget about `recreate_timers` when we add a timer.

Or rather: Should this method maybe be called `start` or `initialize`, and the constructor shouldn't return any outcomes at all?

Reply:
The problem is, this method works a bit differently than the constructor or other places that create timers. For example, for the active validator this creates a timer that will fire immediately, instead of when it _should_ fire, because there is no simple way of determining when that should be (and I'm not sure if that's okay - but I think the active validator will handle that properly?).

Also, since this is supposed to be temporary, I think living with this bit of duplication for now might be preferable (easier to remove it afterwards).

Reply:
OK; just make sure to add the timer for the log message, and let's try to remove the method again as soon as possible.